src/ploneorg/addonlisting/browser/static/w3.css

```diff
@@ -0,0 +1,210 @@
+﻿/* W3.CSS 4.04 Apr 2017 by Jan Egil and Borge Refsnes */
```

Comment:
I was trying to w3.css framework as it has convenient classes for cards,lists etc. https://www.w3schools.com/w3css/default.asp says that it is free to use. No license is necessary.  I tried following the import process from https://docs.plone.org/external/plone.app.dexterity/docs/advanced/static-resources.html and didn't use resourse registeries method because I didn't think it was necessary to available site-wide. 

what would you suggest to be a good way for using custom css with out the risk of breaking the plone theme?